tests/testthat/test-print.R

```diff
@@ -37,6 +37,11 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
     temp_dir <- file.path(get_model_working_directory(MOD1), "print-test")
     fs::dir_create(temp_dir)
     on.exit(fs::dir_delete(temp_dir))
+
+    withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
+      bbi_init(temp_dir, ".", .no_default_version = TRUE)
+    })
```

Comment:
You switched the other spots over to using the dummy file.  Is there are reason this one doesn't use that approach?